lib/prepare_release.js

```diff
@@ -297,6 +297,7 @@ export default class ReleasePreparation {
       '--group',
       '--filter-release',
       '--start-ref',
+      '--markdown',
       this.getLastRef()
```

Comment:
This seems wrong. `this.getLastRef()` is the argument to the `--start-ref` flag.

Reply:
Well, that worked when I tested it. I think since `--markdown` is a bool flag it won't affect the `--start-ref`. 
But, anyway, it's better to move one argument left and make it clear.